edweiss-app/locales/en/todo.json

```diff
@@ -0,0 +1,54 @@
+{
+	"name": {
+		"yet" : "Not Yet",
+		"in_progress": "In Progress",
+		"done": "Finished",
+		"archived": "Archived"
+	},
+
+	"status": {
+		"yet" : "To be done, soon.",
+		"in_progress": "Work in progress ...",
+		"done": "Finished, congrats !",
+		"archived": "Archived, well done !"
+	},
+
+	"todolist_header": "My Todos",
+	"create_header": "Create Todo",
+	"edit_header": "Edit Todo",
+	"name_placeholder": "Daily Workout",
+	"description_placeholder": "Go to the Gym",
+	"save_button": "Save",
+	"edit_button": "Edit",
+	"archive_bg_when_active": "Archive this Todo",
+	"archive_bg_when_archived": "De-archive this Todo",
+	"delete_btn_title": "Delete",
+	"filter_modal_title": "Filter Todos",
+	"close_btn": "Close",
+	"list_empty": "No Todos yet, create one !",
+	"archived_success_toast": "archived !",
+	"unarchived_success_toast": "unarchived !",
+	"archived_success_toast_sub": "You can find it in the archived list.",
+	"unarchived_success_toast_sub": "You can find it in the active list.",
+	"date_btn_title": "Due Date",
+	"time_btn_title": "Time",
+	"today_status": "Today",
+	"tomorrow_status": "Tomorrow",
+	"yesterday_status": "Yesterday",
+	"dued_on": "Due on : ",
+	"at": " at ",
+	"name_label": "Name",
+	"description_label": "Description",
+	"was_added_toast": " was added !",
```

Comment:
Make sure the capitalization is consistent. For example, "archived_success_toast" is lowercase, but "Was added toast" could be capitalized consistently across the file to ensure uniformity.